alerting/build.gradle

```diff
@@ -251,13 +251,13 @@ String bwcVersion = "1.1.0.0"
 String baseName = "alertingBwcCluster"
 String bwcFilePath = "src/test/resources/bwc"
 String bwcOpenSearchPlugin = "opensearch-alerting-" + bwcVersion + ".zip"
-String bwcRemoteFile = 'https://ci.opensearch.org/ci/dbc/bundle-build/1.1.0/20210930/linux/x64/builds/opensearch/plugins/' + bwcOpenSearchPlugin
+String bwcRemoteFile = 'https://ci.opensearch.org/ci/dbc/bundle-build/1.1.0/2.3.0930/linux/x64/builds/opensearch/plugins/' + bwcOpenSearchPlugin
```

Comment:
The BWC workflow seems to be failing because the 1.1 zip isn't available using this modified URL. The old one still has the zip though. This change may need to be reverted.